### DISCUSSION TITLE
Apps layout, how to open automatically apps

### DISCUSSION THREAD

**Discussion by codscino** [rxhanson/RectanglePro-Community#454] [Q&A]
I would like App Layouts to launch the apps too, not just reorganize them once opened manually. Is this possible with rectangle pro or with some workaround/other apps?

**Comment by rxhanson** [comment]
Yes, there's a checkbox in the configuration popover for a layout:

![Untitled](https://github.com/rxhanson/RectanglePro-Community/assets/13651296/c6042930-069a-459f-bb66-6a877ddb1a9c)

**Comment by codscino** [comment]
Thanks a lot, it works perfectly. I just bought the pro license. 

A little feedback, the buy license form opened from the menu bar is laggy

**Reply by rxhanson** [reply]
Thanks! I'm getting ready to push an update for the purchasing library that hopefully will fix this.

**Reply by codscino** [reply]
great!
